news/548.api_change

```diff
@@ -0,0 +1 @@
+OmegaConf.to_container now takes a `structured_config_mode` keyword argument, which replaces the `exclude_structured_configs` argument.
```

Comment:
Since this is in between major releases, document it for people that were not aware of exclude_structured_configs.
Also, remove the news fragment for exclude_structured_configs (if it exists).

Reply:
> Since this is in between major releases, document it for people that were not aware of exclude_structured_configs.

Updated the news fragment in https://github.com/omry/omegaconf/pull/552/commits/b74795b28ce5b5fb89b4ce93630f2d9c9d00a9bf

> Also, remove the news fragment for exclude_structured_configs (if it exists).

Removed in https://github.com/omry/omegaconf/pull/552/commits/05f7ccb83b966a860f6721aedc4df7bb446f6af6